### DISCUSSION TITLE
vant3升级vant4后，step 组件内容显示不正常。

### DISCUSSION THREAD

**Discussion by cxhpy** [youzan/vant#11358] [General]
vant3升级vant4后，step 组件内容显示不正常。

**Comment by chenjiahan** [comment]
麻烦提供个复现问题的 codesandbox 链接哈
https://codesandbox.io/s/vant-3-issue-template-8fuq5o

**Comment by cxhpy** [comment]
https://codesandbox.io/s/vant-3-issue-template-forked-wve8lz?file=/src/App.vue
代码在vant3可以正常显示，升级到vant4后就内容位置不对了.1显示在2的位置，如是类推。搞不懂是什么原因

**Comment by chenjiahan** [comment]
应该是受到 Cell value 样式的影响了，建议在 Cell 下面加一个 `<template #title>` 标签，插入到 title 位置。

![image](https://user-images.githubusercontent.com/7237365/205785936-b2a1dbae-4594-4ee7-a80c-20b3245dd272.png)

Cell v3 -> v4 移除了只设置 value 时自动左对齐的特殊处理，所以会有样式区别，我在文档里标注下。

**Reply by cxhpy** [reply]
感谢解答,问题解决。
